docs/advanced-topics/dockstore-cli/dockstore-cli-faq.rst

```diff
@@ -22,29 +22,50 @@ for more information on these two file transfer sources.
 How do I use the Dockstore CLI on a Mac?
 ----------------------------------------
 
-See `Docker for Mac <https://docs.docker.com/engine/installation/mac/>`__ for installation information.
-
-.. note:: Docker behaves a bit differently on a
-    `Mac <https://docs.docker.com/docker-for-mac/osxfs/#/namespaces>`__ than
-    on a typical Ubuntu machine. By default the only shared volumes are
-    /Users, /Volumes, /tmp, and /private. Note that /var is not a shared
-    directory (and can't be set as one). ``cwltool`` uses your TMPDIR (the
-    env variable) to setup volumes with Docker, which on a Mac can default
-    to a subdirectory of /var. In order to get ``cwltool`` working on your
-    Mac, you need to set your TMPDIR to be under one of the shared volumes
-    in Docker for Mac. You can do this by doing something similar to the
-    following:
-    ::
+.. note:: Our tutorials all assume you are using bash on an Ubuntu machine. Beginning with Mac OS 10.15 Catalina, the default shell has changed from bash to zsh. In almost all cases (especially in our tutorials), syntax between bash-on-Ubuntu, bash-on-Mac, and zsh-on-Mac is the same, but it is possible for slight differences to arise.
 
-        export TMPDIR=/tmp/docker_tmp
+See `Docker for Mac <https://docs.docker.com/engine/installation/mac/>`__ for information on installing Docker for Mac. With the exception of some installation and possible folder permission differences (see next FAQ below), usage of the Dockstore CLI on a Mac is more or less the same as it is on Ubuntu.
```

Comment:
This links to Docker Engine while our quick start guide currently links to a page that will suggest Docker Desktop.

Reply:
We may not have a choice for Mac. While Ubuntu has both choices (Docker Engine unhelpfully called "Docker server" here https://docs.docker.com/engine/install/#server ) I'm operating under the assumption Mac users don't have the choice.